test/drivers/test_suites/syscall_exit_suite/open_x.cpp

```diff
@@ -129,4 +129,60 @@ TEST(SyscallExit, openX_failure)
 
 	evt_test->assert_num_params_pushed(6);
 }
+
+TEST(SyscallExit, openX_create_success)
```

Comment:
i've added this test for `open` but we should add tests for all the other syscalls to avoid issues